### PR TITLE
bpo-44504: Make docstring quotes consistent in Lib/_collections_abc.py

### PR DESCRIPTION
**Pull request by kuzmoyev**
https://bugs.python.org/issue44504

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept this contribution by verifying everyone involved has signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

## Recognized GitHub username

We couldn't find a [bugs.python.org](https://bugs.python.org/) (b.p.o) account corresponding to the following GitHub usernames:

@kuzmoyev

This might be simply due to a missing "GitHub Name" entry in one's b.p.o account settings. This is necessary for legal reasons before we can look at this contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.




You can [check yourself](https://check-python-cla.herokuapp.com/) to see if the CLA has been received.

Thanks again for the contribution, we look forward to reviewing it!

File: Lib/_collections_abc.py [hunk 1]
```diff
@@ -811,7 +811,7 @@ def __getitem__(self, key):
         raise KeyError
 
     def get(self, key, default=None):
-        'D.get(k[,d]) -> D[k] if k in D, else d.  d defaults to None.'
+        'D.get(k[,d]) -> D[k] if k in D else d.  d defaults to None.'
```
Comment (nanjekyejoannah) [inline Lib/_collections_abc.py]: I may be wrong but isn't this a comment? hence looks like punctuation, but am no native English speaker.

Comment (kuzmoyev) [reply Lib/_collections_abc.py]: Hmm. Didn't think about that. English looks too similar to Python. Do you think we should close the PR then?

Comment (kuzmoyev) [reply Lib/_collections_abc.py]: I have rolled back this change. And updated this PR to make docstrings' quotes consistent within the module (which I was going to do in another PR).

**Comment by pganssle** [comment]
@kuzmoyev I believe there is a general policy against these sorts of "homogenizing" changes. Various modules have historically has different choice styles and it's not hurting anything at the moment. New and updated code can / should follow PEP 8, but we don't do codebase-wide cosmetic changes like this.

**Pull request closed without merging**